src/SWP/Bundle/MultiTenancyBundle/EventListener/TenantableListener.php

```diff
@@ -53,12 +53,12 @@ public function __construct(EntityManagerInterface $entityManager, TenantContext
     public function onKernelRequest()
     {
         $tenant = $this->tenantContext->getTenant();
+
         if ($tenant && $tenant->getId()) {
-            $tenantId = $tenant->getId();
             $this->entityManager
                 ->getFilters()
                 ->enable('tenantable')
-                ->setParameter('tenantId', $tenantId);
+                ->setParameter('tenantId', $tenant->getCode());
```

Comment:
Seems weird to set a parameter called 'tenantId' with the result of getCode, if the same object has a getId method